### PR TITLE
Enable memory benchmarks for layers

### PR DESCRIPTION
**Pull request by Czaki**
# Description

Enable benchmarks disabled in #6278

### CODE REVIEW

File: napari/benchmarks/benchmark_image_layer.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def time_refresh(self, n):
         """Time to refresh view."""
         self.layer.refresh()
 
-    def _mem_layer(self, n):
+    def mem_layer(self, n):
         """Memory used by layer."""
         # Disabled because of __sizeof__ bug on the main branch and outdated asizeof in pympler
```
Comment (brisvag) [inline napari/benchmarks/benchmark_image_layer.py]: All these comments are left over from the disabling!

Comment (Czaki) [reply napari/benchmarks/benchmark_image_layer.py]: removed

**Comment by brisvag** [comment]
Actually, where/when was this "`__sizeof__` bug" fixed, so that now this can be reenabled?

**Review by brisvag — APPROVED** [review]
My bad, it was in the linked pr.

**Pull request merged**